autobahn/wamp/uri.py

```diff
@@ -142,6 +142,10 @@ def __init__(self, uri, target, options=None):
 
         :param options: An optional options object
         :type options: None or RegisterOptions or SubscribeOptions
+
+        :param check_types:
+        :type check_types: bool
+
```

Comment:
This should have a description too, i guess (here and below).

Reply:
Added that now